service-tests/dotnetstatus.js

```diff
@@ -9,9 +9,20 @@ const t = new ServiceTester({
 });
 module.exports = t;
 
-t.create('get nuget package status')
-  .get('/gh/Research-Institute/json-api-dotnet-core/master/src/JsonApiDotNetCore/JsonApiDotNetCore.csproj.json')
-  .expectJSONTypes(Joi.object().keys({
-    name: Joi.equal('up-to-date'),
-    value: Joi.equal('true', 'false', 'not found', 'invalid', 'inaccessible')
-  }));
+t
+  .create('get nuget package status')
+  .get('/gh/json-api-dotnet/json-api-dotnet-core/JsonApiDotNetCore.json')
+  .expectJSONTypes(
+    Joi.object().keys({
+      name: Joi.equal('dependencies'),
+      value: Joi.equal(
+        'up to date',
+        'out of date',
+        'not found',
+        'processing',
+        'inaccessible',
+        'invalid',
+        'project not found'
```

Comment:
This seems a bit too broad. How about limiting this to `'up to date', 'out of date', and 'processing'`?

It would be good to include a second test for `'project not found'`.

Reply:
Okay that's fair. Will update the PR tonight